app/containers/markdown/index.tsx

```diff
@@ -142,6 +142,9 @@ class Markdown extends PureComponent<IMarkdownProps, any> {
 	renderText = ({ context, literal }: { context: []; literal: string }) => {
 		const { numberOfLines } = this.props;
 		const defaultStyle = [this.isMessageContainsOnlyEmoji ? styles.textBig : {}, ...context.map(type => styles[type])];
+
+		if (this.isMessageContainsOnlyEmoji) return this.renderEmoji({ literal });
```

Comment:
I don't recall having to change this on my poc. What's the use case needed for this line?